bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -373,23 +373,19 @@ class HasProps extends Backbone.Model
         throw new Error("models must be owned by only a single document")
       else
         @document._notify_attach(@)
```

Comment:
My issue/question is about this line. If `@document == doc` haven't we already attached? Why are we calling `_notify_attach` again?